src/main/java/uk/gov/hmcts/reform/idam/web/strategic/PolicyService.java

```diff
@@ -104,4 +109,38 @@ private boolean checkNoActionsBlockingUser(ResponseEntity<EvaluatePoliciesRespon
         return true;
     }
 
+    /**
+     * Sanitise and return request ips in a list of strings
+     * Examples:
+     * "1.1.1.1" => ["1.1.1.1"]
+     * "51.140.12.192:59286" => ["51.140.12.192"]
+     * "51.140.12.192:59286, 10.97.64.4:59250, 10.97.66.7:57249" => ["51.140.12.192", "10.97.64.4", "10.97.66.7"]
+     * "2001:db8:85a3:8d3:1319:8a2e:370:7348" => ["2001:db8:85a3:8d3:1319:8a2e:370:7348"]
+     * "[2001:db8:85a3:8d3:1319:8a2e:370:7348]:1234" => ["2001:db8:85a3:8d3:1319:8a2e:370:7348"]
+     *
+     * @should returnSanitisedIpAddresses
+     */
```

Comment:
name is a bit ambiguous, maybe sanitiseRequestedIps? get indicates you are getting them from somewhere